### ISSUE TITLE
Support deploying with Rails' asset pipeline

### ISSUE THREAD

**Issue by lazyatom** [tomafro/recap#36]
We aren't sure exactly what the best way of handling this, but it's certainly required to support deploying Rails applications.

**Comment by lazyatom** [comment]
The simplest thing to do is copy what Capistrano is doing - compile the assets on the server. Here's the task from capistrano's `assets.rb` file:

``` ruby
task :precompile, :roles => :web, :except => { :no_release => true } do
  run "cd #{latest_release} && #{rake} RAILS_ENV=#{rails_env} #{asset_env} assets:precompile"
end
```

We don't need to set the `RAILS_ENV`, or handle different release folders.

**Comment by lazyatom** [comment]
Fixed in 34569348c2aa36682ac32f4ce74f0b40b67f0e6d.
